### ISSUE TITLE
No rule to make target install

### ISSUE THREAD

**Issue by fallazc** [herbstluftwm/herbstluftwm#1053]
I am trying to package herbstluftwm 0.9.0 for Slackware but I seems that the install target for the Makefile doesn't exist. The same setup used to work for version 0.7.2. Is there any equivalent to that?

Here is how I attempt to build it:
```
rm -rf $PKG
mkdir -p $TMP $PKG $OUTPUT
cd $TMP
rm -rf $PRGNAM-$VERSION
tar xvf $CWD/$PRGNAM-$VERSION.tar.gz
cd $PRGNAM-$VERSION

make PREFIX=/usr/ MANDIR=/usr/man DOCDIR=/usr/doc$PRGNAM-$VERSION || exit 1
make DESTDIR=$PKG PREFIX=/usr/ MANDIR=/usr/man DOCDIR=/usr/doc$PRGNAM-$VERSION install
```
Thanks

**Comment by The-Compiler** [comment]
With 0.8.0, the build system was changed to cmake - see the [install instructions](https://github.com/herbstluftwm/herbstluftwm/blob/master/INSTALL).

**Comment by fallazc** [comment]
Fixed it, so I had to be in the build folder before running the make install.
I think you should make mention of it in the `Installation steps` section

**Comment by blarz** [comment]
> Fixed it, so I had to be in the build folder before running the make install.
> I think you should make mention of it in the `Installation steps` section

This was indeed a missing step, thank you for reporting this! I fixed it with #1055
